### PR TITLE
Create a default config before starting quick search

### PR DESCRIPTION
**Pull request by nv-braf**
The changes are 109-110 & 336-362, the rest are formatting fixes.

This did not end up perturbing any unit tests, as get_configs() is never called from any unit testing. Not sure if we need unit testing coverage for this, we have coverage in the L0_quick_test.

### CODE REVIEW

File: model_analyzer/config/generate/quick_run_config_generator.py [hunk 1]
```diff
@@ -329,8 +333,33 @@ def _get_next_perf_analyzer_config(self, model_variant_name: str,
...
+
+        for model in self._models:
+            default_model_config = BaseModelConfigGenerator.make_model_config(
+                param_combo={},
+                model=model,
+                model_variant_name_manager=self._model_variant_name_manager)
+
+            default_perf_analyzer_config = PerfAnalyzerConfig()
+            default_perf_analyzer_config.update_config_from_profile_config(
+                default_model_config._model_config.name, self._config)
```
Comment (tgerdesnv) [inline model_analyzer/config/generate/quick_run_config_generator.py:347]: don't reach in to private variable.

default_model_config.get_field('name')

**Review by tgerdesnv — CHANGES_REQUESTED** [review]

File: model_analyzer/config/generate/quick_run_config_generator.py (hunk 1 above)
Comment (nv-braf) [reply model_analyzer/config/generate/quick_run_config_generator.py]: Fixed

**Comment by tgerdesnv** [comment]
Did no tests fail from this change?

**Comment by nv-braf** [comment]
> 

That is correct. See the description.

**Review by tgerdesnv — APPROVED** [review]

**Pull request merged**